filament/backend/src/opengl/gl_headers.h

```diff
@@ -115,6 +115,13 @@
 
 #endif
 
+// This is an odd duck function that exists in WebGL 2.0 but not in OpenGL ES.
+#if defined(__EMSCRIPTEN__)
```

Comment:
if you're using __EMSCRIPTEN__ here, I think you should use __EMSCRIPTEN__ in OpenGLDriver.cpp where you're using `glGetBufferSubData`. It would be cleaner.